app/src/pages/200/RequestSubmitted.tsx

```diff
@@ -0,0 +1,36 @@
+import Container from '@material-ui/core/Container';
+import Box from '@material-ui/core/Box';
+import React from 'react';
+import { mdiCheck } from '@mdi/js';
+import Icon from '@mdi/react';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const RequestSubmitted = () => {
+  return (
+    <Container>
+      <Box pt={6} textAlign="center">
+        <Icon path={mdiCheck} size={2} color="#4caf50" />
+        <h1>Access Request Submitted</h1>
+        <Typography>Your access request has been submitted for review.</Typography>
+        <Box pt={4}>
+          <Button
+            onClick={async () => {
+              window.location.href =
```

Comment:
The logout stuff we can refine separately, so I'd just leave this as is, or just stub it out for now.  Based on some conversations in some links Roland sent me, its possibly more involved than just calling a link (unless they've since fixed it).  But ya, eventually it would be good to add a logout function to the useKeycloakWrapper.